src/zope/app/basicskin/standardmacros.py

```diff
@@ -15,16 +15,21 @@
 
 The macros are drawn from various different page templates.
 """
+from __future__ import print_function, absolute_import, division
+
```

Comment:
Perhaps not, but it seems to me it's proactive to ensure that Py2 and Py3 behave as similarly as possible. It's similar to the same reason that one would write `class Thing(object):`, to be sure there are new-style classes everywhere.

The file was being touched now with the express intent of making it work under Python 3, so I didn't see any reason to wait. Especially `print_function` tends to creep in during debugging, and it's a hassle to add and remove it.

But if you feel strongly I can remove it.